edge/pkg/edgehub/process.go

```diff
@@ -53,26 +68,34 @@ func (eh *EdgeHub) dispatch(message model.Message) error {
 		md = modules.MetaGroup
 	case messagepkg.UserGroupName:
 		md = modules.BusGroup
-	default:
-		klog.Warningf("msg_group not found")
-		return fmt.Errorf("msg_group not found")
 	}
 
-	isResponse := eh.isSyncResponse(message.GetParentID())
+	isResponse := isSyncResponse(message.GetParentID())
 	if isResponse {
-		beehiveContext.SendResp(message)
+		beehiveContext.SendResp(*message)
 		return nil
 	}
 	if group == messagepkg.UserGroupName && message.GetSource() == "router_eventbus" {
-		beehiveContext.Send(modules.EventBusModuleName, message)
+		beehiveContext.Send(modules.EventBusModuleName, *message)
 	} else if group == messagepkg.UserGroupName && message.GetSource() == "router_servicebus" {
-		beehiveContext.Send(modules.ServiceBusModuleName, message)
+		beehiveContext.Send(modules.ServiceBusModuleName, *message)
 	} else {
-		beehiveContext.SendToGroup(md, message)
+		beehiveContext.SendToGroup(md, *message)
 	}
 	return nil
 }
 
+func (eh *EdgeHub) dispatch(message model.Message) error {
+	// handler for msg.
+	isProcessed, err := msghandler.ProcessHandler(message, eh.chClient)
+	if isProcessed {
+		return err
```

Comment:
```suggestion
		return nil
```
?

Reply:
if been processed, we need to return nil if no-error, return error if error happens, so just return `err`, and it contains nil or error circumanstances.

Reply:
If err != nil,  dispatch msg should be failed.